crates/bevy_pbr/src/render/mesh_preprocess.wgsl

```diff
@@ -43,20 +74,103 @@ struct PreprocessWorkItem {
 // The output array of `Mesh`es.
 @group(0) @binding(3) var<storage, read_write> output: array<Mesh>;
 
+#ifdef INDIRECT
+// The array of indirect parameters for drawcalls.
+@group(0) @binding(4) var<storage, read_write> indirect_parameters: array<IndirectParameters>;
+#endif
+
+#ifdef FRUSTUM_CULLING
+// Data needed to cull the meshes.
+//
+// At the moment, this consists only of AABBs.
+@group(0) @binding(5) var<storage> mesh_culling_data: array<MeshCullingData>;
+
+// The view data, including the view matrix.
+@group(0) @binding(6) var<uniform> view: View;
+
+// Returns true if the view frustum intersects a sphere, specified in world
```

Comment:
This means a sphere is inside the frustum right? At first I interpreted "intersects" as if it's at an intersection between outside and inside the frustum so just making sure.

Reply:
It means that there's at least one point that's simultaneously inside the sphere and inside the frustum.

Reply:
Yeah, makes sense. That's what I assumed, but wasn't completely sure. I originally read it as intersecting one of the frustum plane instead of being inside the volume of the frustum. Anyway, you can resolve this.